runners/pom.xml

```diff
@@ -33,6 +33,7 @@
   <name>Apache Beam :: Runners</name>
 
   <modules>
+    <module>core-construction-java</module>
     <module>core-java</module>
```

Comment:
Separate PR; I agree with the move, but think that one probably deserves some mailing list discussion.